app/views/user_mailer/claim_notification.html.erb

```diff
@@ -3,32 +3,37 @@
 
-  <p>Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please <%= support_link('contact AO3 Support') %>.</p>
+  <p><%= t(".access.html", contact_support_link: support_link(t(".access.contact_support"))) %></p>
 
-  <p>These works were written under the e-mail: <%= @external_email %></p>
+  <p><%= t(".works_by", email: @external_email) %></p>
 
   <ul>
     <% @claimed_works.each do |work| %>
-      <li><%= style_link(work.title.html_safe, work_url(work)) %>
-        <%= (work.fandom_string.blank? ? "" : " (#{work.fandom_string})") %></li>
+      <li>
+        <% if work.fandom_string.present? %>
+          <%= t(".work_info.html", work_link: style_link(work.title, work_url(work)), fandom: work.fandom_string.split(",").to_sentence) %>
+        <% else %>
+          <%= style_link(work.title, work_url(work)) %>
+        <% end %>
+      </li>
     <% end %>
   </ul>
 
-  <p>To preserve rec lists and bookmarks, the imported archive's web addresses may redirect to the imported copy of these works for a limited time (check the announcement post for your archive to be sure). If you've already uploaded a copy of these works and you did NOT use the import from URL feature, there will be two copies of the same work on the archive.</p>
+  <p><%= t(".redirects") %></p>
 
-  <p>If you would like Open Doors to update the redirect to point to your pre-existing work, please delete the imported copy, and <%= style_link('contact Open Doors', 'https://opendoors.transformativeworks.org/en/contact-open-doors/') %> with your AO3 account name, your account name on the imported archive, and the title and URL of the fanwork you would like the redirect to point to. (If you have multiple works you would like to change the redirects for, you can list these in one email.)</p>
+  <p><%= t(".update_redirect.html", contact_open_doors_link: opendoors_link(t(".update_redirect.contact_open_doors"))) %></p>
 
-  <p>If you had other works on the imported archive under an e-mail address you can no longer access, please <%= style_link('contact Open Doors', 'https://opendoors.transformativeworks.org/en/contact-open-doors/') %> with any information that can help verify your identity.</p>
+  <p><%= t(".other_works.html", contact_open_doors_link: opendoors_link(t(".other_works.contact_open_doors"))) %></p>
 
-  <p>For other inquiries, please <%= support_link('contact AO3 Support') %>.</p>
+  <p><%= t(".questions.html", contact_support_link: support_link(t(".questions.contact_support"))) %></p>
 
-  <p>If you're contacting us, please add email addresses from @transformativeworks.org to your list of safe contacts and check your spam folders for our reply.</p>
+  <p><%= t ".email_tips" %></p>
```

Comment:
One minor thing! Could you change this to `<%= t(".email_tips") %>` to match the others?